rocm/ubi9-python-3.9/Dockerfile

```diff
@@ -15,6 +15,7 @@ LABEL name="odh-notebook-rocm-python-3.9" \
 USER 0
 WORKDIR /opt/app-root/bin
 
+# Please keep in sync with jupyter/rocm/pytorch/ubi9-python-3.9/Pipfile and runtimes/rocm-pytorch/ubi9-python-3.9/Pipfile
```

Comment:
Can we make these comments more generic? 
Something like `Please keep the version in sync with other ROCm/Python3.9 based images`.
Otherwise, wouldn't we need to list the folders for tensorflow too? It could be annoying to maintain them.

I also wonder if we should add these comments at all. 
I believe there could be a case where we would use different ROCm versions for Pytorch and TF, as already happened in the first place?

Reply:
> Can we make these comments more generic?

Yeah, I can update it to be more generic.

> I believe there could be a case where we would use different ROCm versions for Pytorch and TF, as already happened in the first place?

Well, at least until we won't use different Dockerfile for each, where the ROCm is installed in the first place, we should keep it in sync, otherwise different ROCm versions are installed from rpms and python packages... that is why I raised this to make it same version in both.

Reply:
Updated